bukuserver/README.md

```diff
@@ -174,6 +175,27 @@ a technique used to identify and track a person's web browsing habits.
 
 It is important to note that favicon can potentially be exploited in this way.
 
+### API
+
+Bukuserver implements a RESTful API that provides an HTTP-based interface for the main functionality.
+
+The API root is at `/api`; you can also access a [Swagger](https://swagger.io/tools/swagger-ui)-based interactive doc at the endpoint `/apidocs` (e.g. `http://localhost:5000/apidocs`).
```

Comment:
I haven't added any links to the app UI; kinda doubt anyone who didn't check the README would care about such a thing :sweat_smile: